app/flask_api_client/api_client.py

```diff
@@ -0,0 +1,72 @@
+import requests
+from flask import json
+from app.model import User
```

Comment:
Should be a relative import. Also, in other modules we are separating imports from 3rd party libraries from application imports with a blank line.